homework-4/style.css

```diff
@@ -0,0 +1,175 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+a {
+  text-decoration: none;
+}
+button {
+  border: none;
+  background: transparent;
+}
+.container {
+  max-width: 1440px;
+  margin: 0 auto;
+}
+.d-flex{
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 30px;
+  padding: 0 40px;
+  margin-top: 40px;
+  color: #7f7f7f;
+}
+body {
+  padding: 0;
+  background: #fff;
+  font-family: 'Arial', 'Helvetica', sans-serif;
+  font-size: 20px;
+}
+.logo,
+.my-account,
+.login {
+  color: #7f7f7f;
+}
+.my-account {
+  padding-right: 10px;
+  border-right: 1px solid #e5e5e5;
+}
+.login {
+  padding-left: 10px;
+}
+.filter {
+  padding: 20px 40px 0 40px;
+  color: #7f7f7f;
+  font-size: 16px;
+}
+.main-title {
+  padding: 30px 0 30px 50px;
+  font-size: 25px;
+  background-color: #3f48cc;
+  color: #fff;
+}
+.edit-input {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.users-icon {
+  margin-right: 20px;
+  font-size: 30px;
+}
+.filter-form {
+  display: flex;
+  justify-content: space-between;
+  margin: 20px 0;
+}
+.filter-button {
+  padding: 10px 20px;
+  background-color: #3f48cc;
+  color: #fff;
+}
+.input {
+  padding: 10px 20px;
+  margin-right: 20px;
+}
+.icon-plus {
+  color: #fff;
+  background-color: #ed1e64;
+  border-radius: 50%;
+  padding: 20px;
+}
+.icon-close {
+  padding: 5px 7px;
+  background-color: #e5e5e5;
+  border-radius: 50%;
+  color: #fff;
+}
+.icon-pen {
+  margin-right: 20px;
+  color: #e5e5e5;
+  font-size: 18px;
+}
+.add-new-customer {
+  margin-left: 400px;
+}
+.icon-close-btn {
+  display: none;
+  margin-left: 38px;
+  padding: 5px 7px;
+  background-color: #e5e5e5;
+  border-radius: 50%;
+  color: #fff;
+}
+.table {
+  position: relative;
+  border-spacing: 25px 25px;
+  border-collapse: inherit;
+  border-top: 1px solid #e5e5e5;
+  max-width: 1440px;
```

Comment:
you have same max-width for container. No need to add it here. If something will go wrong - you can write `max-width: 100%;`

Reply:
fixed